config/peerpods/podvm/Dockerfile.podvm-oci

```diff
@@ -0,0 +1,6 @@
+FROM scratch
+
+ARG PODVM_IMAGE_SRC
+ENV PODVM_IMAGE_PATH="/image/podvm.qcow2"
+
+COPY $PODVM_IMAGE_SRC $PODVM_IMAGE_PATH
```

Comment:
We consider the image to be built with the pause image if needed right? in the future we may want allow embedding it at build, if possible..

Reply:
ahh that's a good point, I have never thought about it, I will explore a bit on this and come back.. but do you think we can do this change on the next iteration (another PR) or would you like to see it in this PR itself..

Reply:
yet, I think we can do it later